index.js

```diff
@@ -0,0 +1,91 @@
+var analyze = require('./lib/analyze-raptor');
+var Promise = require('promise');
+var fs = require('fs');
+
+var metric = 'visuallyLoaded';
+
+var readFile = function(dataFile, appContext, appEntryPoint) {
+  console.log('Processing %s', dataFile);
```

Comment:
Remove all `console.log`s that do not log data. If you'd like, you can use the `debug` module to log this information.